WordPress/Classes/ViewRelated/NUX/SignupEpilogueCell.swift

```diff
@@ -37,7 +37,7 @@ class SignupEpilogueCell: UITableViewCell {
     private let passwordTopMargin: CGFloat = 16
 
     private var cellType: EpilogueCellType?
-    open var delegate: SignupEpilogueCellDelegate?
+    open weak var delegate: SignupEpilogueCellDelegate?
```

Comment:
There is a retain cycle between the `SignupEpilogueTableViewController` and this `SignupEpilogueCell`.

https://github.com/wordpress-mobile/WordPress-iOS/blob/48b8a391394f2b4aee9b886abe21f62a521c0a81/WordPress/Classes/ViewRelated/NUX/SignupEpilogueTableViewController.swift#L214